CMakeLists.txt

```diff
@@ -67,8 +67,8 @@ endif(  )
 add_executable( secvarctl ${SRC} )
 
 #no crypto means don't compile the generate command = smaller executable
-option( NO_CRYPTO "Build without crypto functions for smaller executable, some functionality lost" OFF )
-if ( NOT NO_CRYPTO )
+option( CRYPTO_READ_ONLY "Build without crypto write functions for smaller executable, some functionality lost" OFF )
+if ( NOT CRYPTO_READ_ONLY )
```

Comment:
I like this a lot better. :+1: 

Side question: it doesn't matter whether we use either crypto lib for this right?
Side side question: does this work for both static and dynamic linked builds?
Side side side question: can we build static with all crypto libs we support, or is that limited to mbedtls only?

None of the above questions are relevant to this PR, just wondering for future.